README.md

```diff
@@ -9,53 +9,47 @@
 `ember-notify` is compatible with the following presentation frameworks:
 
 - Zurb Foundation 6 (default)
-- Zurb Foundation 5: `{{ember-notify messageStyle='foundation-5'}}`
-- Thoughtbot Refills: `{{ember-notify messageStyle='refills'}}`
-- Twitter Bootstrap: `{{ember-notify messageStyle='bootstrap'}}`
-- Semantic-UI: `{{ember-notify messageStyle='semantic-ui'}}`
-- UIKit: `{{ember-notify messageStyle='uikit'}}`
+- Zurb Foundation 5: `<EmberNotify @messageStyle="foundation-5" />`
+- Thoughtbot Refills: `<EmberNotify @messageStyle="refills" />`
+- Twitter Bootstrap: `<EmberNotify @messageStyle="bootstrap" />`
+- Semantic-UI: `<EmberNotify @messageStyle="semantic-ui" />`
+- UIKit: `<EmberNotify @messageStyle="uikit" />`
 
 The CSS animations are inspired by CSS from [alertify.js](http://fabien-d.github.io/alertify.js/). You can also customize the positioning and animations by overriding the default `ember-notify` CSS class. For usage, see the [animations example](#custom-animations).
 
 ## Usage
 
-1. Add `{{ember-notify}}` to one of your templates, usually in `application.hbs`
+1. Add `<EmberNotify />` to one of your templates, usually in `application.hbs`
 2. Inject the `notify` service
 3. Display messages using the `info`, `success`, `warning`, `alert` and `error` methods
 
 ### Examples
 
 ```js
-import Component from '@ember/component';
+import Component from '@glimmer/component';
 import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
 
-export default Component.extend({
-  notify: service(),
+export default class MyComponent extends Component {
+  @service notify;
 
-  actions: {
-    sayHello() {
-      this.notify.info('Hello there!');
-    }
+  @action
+  sayHello() {
+    this.notify.info('Hello there!');
   }
-});
+}
 ```
 
 By default the notifications close after 2.5 seconds, although you can control this in your template:
 
-```handlebars
-{{ember-notify closeAfter=4000}}
-```
-
-Using angle bracket invocation, available in Ember 3.4+
-
 ```handlebars
 <EmberNotify @closeAfter={{4000}} />
 ```
 
 Or you can control when each message is closed:
 
 ```js
-let message = this.notify.alert(`You can control how long it's displayed`, {
+let message = this.notify.alert('You can control how long it's displayed', {
```

Comment:
This is not valid `js` because there's a `i` in the sentence

Reply:
```suggestion
let message = this.notify.alert('You can control how long it is displayed', {
```